packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,28 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const noPublishDateContentTypes = ['event', 'webinar', 'whitepaper', 'contact'];
 
 <endeavor-breadcrumbs-website-section section=content.primarySection />
 <cms-content-name tag="h1" class="page-wrapper__title" obj=input.content />
 <cms-content-teaser tag="p" class="page-wrapper__deck" obj=input.content />
+<if(content.type === 'event')>
+  <p class="page-wrapper__event-date">
+    <endeavor-content-starts obj=input.content />
+    <if(content.ends)>
+      <span class="page-wrapper__event-date-divider"> - </span>
```

Comment:
I'm still confused by what is happening here... You shouldn't need to wrap this in a `<span>` and, rather, can control the display using nothing but pure CSS.

Reply:
k